.circleci/config.yml

```diff
@@ -660,4 +689,10 @@ jobs:
           at: ~/
       - run-smoke-tests:
           minimal: true
+  publish-cloudsmith:
+    executor: cloudsmith/default
+    steps:
+      - attach_workspace:
+          at: ~/
+      - publish-cloudsmith
```

Comment:
30+ minutes to publish the packages seems like a long time. Is is because of large packages and slow upload speeds? :D

Reply:
Not really sure where the bottleneck is, but yeah, it really does seem to take that long.  I could split it into RPM and Deb but I was trying to avoid another big attach that eats time.

Reply:
Ack.

We could consider using the [small resource class](https://circleci.com/docs/2.0/configuration-reference/#resource_class) here if it doesn't affect the performance too much - that would make these runs less costly.

Reply:
Good point.  It shouldn't need bigger, it's all network operations really.